calendar/inc/class.calendar_hooks.inc.php

```diff
@@ -223,6 +223,11 @@ static function settings($hook_data)
 			$export_tzs += Api\DateTime::getTimezones();
 		}
 		$link_title_options = calendar_bo::get_link_options();
+		$whole_day_behaviour_options = array(
+			'all_appointments' => lang('All as one-liners'),
+			'except_me'        => lang('All as one-liners except mine'),
+			'default'          => lang('Non-blocking as one-liners')
```

Comment:
The default should be "" AND the first option, so nothing changes for existing installs!